contents/docs/libraries/next-js.md

```diff
@@ -7,37 +7,41 @@ PostHog makes it easy to get data about traffic and usage of your [Next.js](http
 
 This guide walks you through integrating PostHog into your Next.js app using the [React](/docs/libraries/react) and the [Node.js](/docs/libraries/node) SDKs.
 
-You can see a working example of this integration in our [Next.js demo app](https://github.com/PostHog/posthog-js/tree/master/playground/nextjs)
+> You can see a working example of this integration in our [Next.js demo app](https://github.com/PostHog/posthog-js/tree/master/playground/nextjs)
 
-Next.js has both client and server-side rendering. We'll cover both in this guide.
+Next.js has both client and server-side rendering as well as pages and app routers. We'll cover all of these options in this guide.
 
 ## Prerequisites
 
 To follow this guide along, you need:
 
-1. a PostHog account (either [Cloud](/docs/getting-started/cloud) or [self-hosted](/docs/self-host))
-2. a running Next.js application
+1. A PostHog instance (either [Cloud](https://app.posthog.com/signup) or [self-hosted](/docs/self-host))
+2. A Next.js application
 
-## Client-side analytics
+## Client-side setup (analytics, pageviews, feature flags)
```

Comment:
For the text in the brackets, It's not super clear what they mean or why they're in brackets.
I think it would be better to have a short component `Which features are available?`, like we do in our SDK docs ([example](https://posthog.com/docs/libraries/js))